static/js/util/validation.js

```diff
@@ -11,14 +11,16 @@ import type { UIState } from '../reducers/ui';
 import { filterPositiveInt } from './util';
 import { HIGH_SCHOOL, EDUCATION_LEVELS } from '../constants';
 
-let handleNestedValidation = (profile, keys, nestedKey) => {
+let handleNestedValidation = (profile: any, keys, nestedKey) => {
```

Comment:
actually got it to work! adding `nestedKey: string` gets `profile: Profile` to work correctly.